tests/uefi_sb/test_cert_inheritance.py

```diff
@@ -265,7 +265,7 @@ def keys_auths_for_joined_host(self, host, hostB1):
         from packaging import version
         version_str = "8.3"
         if not hostB1.xcp_version >= version.parse(version_str):
-            pytest.skip(f"This test requires a second XCP-ng pool with version >= {version_str}")
+            pytest.fail(f"This test requires a second XCP-ng pool with version >= {version_str}")
```

Comment:
`pytest.fail` says a correct test detected an error in the product. To show it as an error in the test it should raise an exception instead.